src/comment_worker.py

```diff
@@ -915,9 +916,9 @@ def tax(self, sess, comment, investor, tax_temp):
             first()
 
         # Defining some tax limits, so
-        # 5 <= firm.tax <= 75
-        lower = 5
-        upper = 75
+        # 0 <= firm.tax <= 100
```

Comment:
We initially implemented the limits, so firms won't be able to scam its members. As history showed, we can get rid of this too.